src/coreclr/tools/GCLogParser/parse-hb-log.cs

```diff
@@ -495,59 +501,59 @@ static void PassZero(string strLog)
         // start running and how active they are.
         // Cleared every GC.
-        static List<int> threadsSeenPerTimeUnit = new List<int>(56);
-        static Dictionary<int, int> threadsSeen = new Dictionary<int, int>(56);
-        static Dictionary<int, int> threadsSeenTotal = new Dictionary<int, int>(56);
-        static List<int> threadsToPrint = new List<int>(8);
-        static string strThreadIndices = null;
-        static int gcIndexToPrintStart = -1;
-        static int gcIndexToPrintEnd = 1000000;
-        static string strGCRange = null;
-        static int totalGCCount = 0;
-        static int totalGCDurationMS = 0;
-        static int totalAllocMB = 0;
-        static int totalBudgetMB = 0;
+        private static List<int> threadsSeenPerTimeUnit = new List<int>(56);
+        private static Dictionary<int, int> threadsSeen = new Dictionary<int, int>(56);
+        private static Dictionary<int, int> threadsSeenTotal = new Dictionary<int, int>(56);
+        private static List<int> threadsToPrint = new List<int>(8);
+        private static string strThreadIndices = null;
+        private static int gcIndexToPrintStart = -1;
+        private static int gcIndexToPrintEnd = 1000000;
+        private static string strGCRange = null;
+        private static int totalGCCount = 0;
+        private static int totalGCDurationMS = 0;
+        private static int totalAllocMB = 0;
+        private static int totalBudgetMB = 0;
 
         // This represents allocated MB on each heap; parse from the lines following [GC_alloc_mb]
-        static int[] AllocMB;
-        static int budgetMB = 0;
+        private static int[] AllocMB;
+        private static int budgetMB = 0;
         // These are subscript chars for 0-9.
-        static char[] unicodeChars = { '\x2080', '\x2081', '\x2082', '\x2083', '\x2084', '\x2085', '\x2086', '\x2087', '\x2088', '\x2089' };
-        static string[] passOneFileTypes = { "thread", "alloc" };
+        private static char[] unicodeChars = ['\x2080', '\x2081', '\x2082', '\x2083', '\x2084', '\x2085', '\x2086', '\x2087', '\x2088', '\x2089'];
+        private static string[] passOneFileTypes = ["thread", "alloc"];
 
-        static void PrintToAllPassOneFiles(string strLine)
+        private static void PrintToAllPassOneFiles(string strLine)
         {
             for (int fileIndex = 0; fileIndex < (int)PassOneViewType.MaxType; fileIndex++)
             {
                 swPassOneFiles[fileIndex].WriteLine(strLine);
             }
         }
 
-        static void PrintAllocToAllPassOneFiles()
+        private static void PrintAllocToAllPassOneFiles()
         {
             string strAlloc = string.Format("{0,6}", nodeIndexToPrint);
             int procStart = procsPerNode * nodeIndexToPrint;
             int procEnd = procsPerNode * (nodeIndexToPrint + 1);
 
-            int currentGCAllocMBAllHeaps = 0;
+            // int currentGCAllocMBAllHeaps = 0;
```

Comment:
assuming these were unused locals, should the lines be deleted?

Reply:
Yup, I was getting warning (unnecessary assignments etc.).